scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -201,8 +168,27 @@ task Filter {
         GATK_JAR=${gatk4_jar_override}
     fi
 
-  	java -Xmx4g -jar $GATK_JAR FilterMutectCalls -V ${unfiltered_calls} -O ${output_prepend}-filtered.vcf \
-       ${"-contaminationTable " + contamination_table}
+    if [[ "${variants_for_contamination}" == *.vcf ]]; then
+        java -Xmx4g -jar $GATK_JAR GetPileupSummaries -I ${tumor_bam} -L ${intervals} -V ${variants_for_contamination} -O pileups.table
```

Comment:
I think that a good compromise would be to put Sequencing artifacts as its own task but leave the rest, which are very quick, in one task.